docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```diff
@@ -33,6 +33,18 @@ public DiagnosticsClient
         Guid profilerGuid,
         string profilerPath,
         byte[] additionalData = null);
+        
```

Comment:
I believe we also added [WriteDumpAsync](https://github.com/dotnet/diagnostics/blob/27172ce4d05e8a3b0ffdefd65f073d40a1b1fe54/src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs#L157-L162)